distribution/docker/druid.sh

```diff
@@ -196,4 +198,14 @@ then
     mkdir -p ${DRUID_DIRS_TO_CREATE}
 fi
 
+# Check if the other configuration directorie variable is set and if it is, copy the files from there
+if [ -n "$DRUID_ADDITIONAL_CONF_DIR" ]
+then
+    if [  -d "$DRUID_ADDITIONAL_CONF_DIR" ]
+    then
+        cp $DRUID_ADDITIONAL_CONF_DIR/* $COMMON_CONF_DIR
+    fi
+fi
+
+
 exec bin/run-java ${JAVA_OPTS} -cp $COMMON_CONF_DIR:$SERVICE_CONF_DIR:lib/*: org.apache.druid.cli.Main server $@
```

Comment:
why not take an env such as "EXTRA_CLASSPATH" and then add it here in the command line? No need to copy files and users can pass multiple directories if needed.

Reply:
good point, will fix